application/src/main/java/org/togetherjava/tjbot/commands/tags/TagSystem.java

```diff
@@ -54,19 +67,41 @@ static Button createDeleteButton(String componentId) {
      *
      * @param id the id of the tag to check
      * @param event the event to send messages with
+     * @param componentIdGenerator the component id generator to generate the button ids with
+     * @param userToReplyTo the user that was originally meant to be replied to when the tag command
+     *        was invoked
      * @return whether the given tag is unknown to the system
      */
-    boolean handleIsUnknownTag(String id, IReplyCallback event) {
+    boolean handleIsUnknownTag(String id, IReplyCallback event,
+            ComponentIdGenerator componentIdGenerator, @Nullable OptionMapping userToReplyTo) {
```

Comment:
NIT: same naming issue with the reply user. maybe `replyTargetUser`. But `userToReplyTo` is also okay, just sounds a bit wordy.